MapboxNavigation/GenericRouteShield.swift

```diff
@@ -98,9 +98,10 @@ public class GenericRouteShield: StylableView {
         let proxy = GenericRouteShield.appearance()
         var backgroundColor = proxy.backgroundColor
         var foregroundColor = proxy.foregroundColor
+        let resolvedColorSelector = Selector(("resolvedColorWithTraitCollection:" as NSString) as String)
         if #available(iOS 13.0, *),
-            let bgColor = backgroundColor, bgColor.responds(to: #selector(UIColor.resolvedColor(with:))),
-            let fgColor = foregroundColor, fgColor.responds(to: #selector(UIColor.resolvedColor(with:))),
+            let bgColor = backgroundColor, bgColor.responds(to: resolvedColorSelector),
+            let fgColor = foregroundColor, fgColor.responds(to: resolvedColorSelector),
             let currentTraitCollection = UIApplication.shared.keyWindow?.traitCollection {
             backgroundColor = bgColor.resolvedColor(with: currentTraitCollection)
```

Comment:
`resolvedColor(with:)` is still being called directly:

```
/Users/distiller/project/MapboxNavigation/GenericRouteShield.swift:106:31: error: value of type 'UIColor' has no member 'resolvedColor'
            backgroundColor = bgColor.resolvedColor(with: currentTraitCollection)
                              ^~~~~~~ ~~~~~~~~~~~~~
/Users/distiller/project/MapboxNavigation/GenericRouteShield.swift:107:31: error: value of type 'UIColor' has no member 'resolvedColor'
            foregroundColor = fgColor.resolvedColor(with: currentTraitCollection)
                              ^~~~~~~ ~~~~~~~~~~~~~
/Users/distiller/project/MapboxNavigation/ExitView.swift:156:31: error: value of type 'UIColor' has no member 'resolvedColor'
            backgroundColor = bgColor.resolvedColor(with: currentTraitCollection)
                              ^~~~~~~ ~~~~~~~~~~~~~
/Users/distiller/project/MapboxNavigation/ExitView.swift:157:31: error: value of type 'UIColor' has no member 'resolvedColor'
            foregroundColor = fgColor.resolvedColor(with: currentTraitCollection)
                              ^~~~~~~ ~~~~~~~~~~~~~
```

Pass the selector into `perform(_:with:afterDelay:)` instead.